mobify/platform-scaffold
# WEB-1434  view order page

This PR adds the view order page to the scaffold. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1434
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/806

## Changes
- Added account-view-order template
- Added init account view order commands to connectors
- Styled account-view-order template

## Todos

## How to test-drive this PR
- checkout the `view-order-page-commands` branch of the SDK
- cd into the SDK folder
- run `npm link`
- cd back into the scaffold folder
- run `npm link progressive-web-sdk`
- run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Sign in to an account with orders
- Navigate to the order list page
- Click "view" for one of the orders in the list
- the page should work as expected
- switch to the SFCC connector and preview the SFCC site
- the page should still work as expected
